folium/folium.py

```diff
@@ -16,6 +16,8 @@
 import json
 from uuid import uuid4
 
+import imageio
```

Comment:
There will be (probably) other failures that are unrelated.  That is something I have on my plate to fix.  Ignore them for now...

BTW add `imageio` to our `.travis.yml`!